code/modules/shuttle/computer.dm

```diff
@@ -305,9 +305,16 @@
 					to_chat(user, SPAN_NOTICE("[src]'s screen says \"Unable to launch, fuel insufficient\"."))
 					return
 
-				if(tgui_alert(user, "Early launch the lifeboat?", "Confirm", list("Yes", "No"), 10 SECONDS) == "Yes")
-					to_chat(user, SPAN_NOTICE("[src]'s screen blinks and says \"Early launch accepted\"."))
-					lifeboat.evac_launch()
+				var/response = tgui_alert(user, "Early launch the lifeboat?", "Confirm", list("Yes", "No", "Instantly"), 10 SECONDS)
+				switch(response)
+					if("Yes")
+						to_chat(user, "[src]'s screen blinks and says \"Early launch accepted\".")
+						shipwide_ai_announcement("Launch command received. " + (lifeboat.id == MOBILE_SHUTTLE_LIFEBOAT_PORT ? "Port" : "Starboard") + " lifeboat doors will close in 10 seconds.")
+						addtimer(CALLBACK(lifeboat, TYPE_PROC_REF(/obj/docking_port/mobile/crashable/lifeboat, evac_launch)), 10 SECONDS)
+					else if("Instantly")
```

Comment:
```suggestion
					if("Instantly")
```
It's a switch, else is redundant (and probably should break the code).

Reply:
yeah I ended up adding that to fix an error and then it worked locally but not with linter... anyways thanks, resolved